### DISCUSSION TITLE
simpleError in allLSI[cn, , drop = FALSE]: subscript out of bounds

### DISCUSSION THREAD

**Discussion by wshands** [GreenleafLab/ArchR#926] [General]
I'm seeing this error and not sure how to fix it. Do you have any suggestions?
Thanks!

```
************************************************************
2021-07-22 23:33:05 : ERROR Found in .computeLSI for
LogFile = ArchRLogs/ArchR-addIterativeLSI-2366a2c0a12-Date-2021-07-22_Time-23-32-50.log

<simpleError in allLSI[cn, , drop = FALSE]: subscript out of bounds>

2021-07-22 23:33:05 : errorList, Class = list

errorList$mat: nRows = 24378, nCols = 53
errorList$mat: NonZeroEntries = 48946, EntryRange = [ 2.96032606834252 , 11.7943454715497 ]
5 x 5 sparse Matrix of class "dgCMatrix"
```

[ArchR-addIterativeLSI-2366a2c0a12-Date-2021-07-22_Time-23-32-50.log](https://github.com/GreenleafLab/ArchR/files/6866012/ArchR-addIterativeLSI-2366a2c0a12-Date-2021-07-22_Time-23-32-50.log)

**Comment by rcorces** [comment]
Please post bugs/errors to the Issues section and use the provided Issue Templates. 
That being said, from what you've provided I will not have enough information to help you.

**Reply by wshands** [reply]
It seems this may have happened due to a substandard data set.

**Reply by ReshmaRamaiah10** [reply]
Hi @wshands! Hope you are doing well.
I'm running into the same error for a few samples. I was wondering how do you troubleshoot this error?

**Reply by BDkokishinbara** [reply]
Hi, I also encountered same problem, but I solve it.
My sample name was set as "AT209_72h_fresh", and the error occurred, but the name "AT_fresh" is OK.
Probably, the number of name is not good for addIterativeLS.
